sea-orm-macros/src/derives/active_enum.rs

```diff
@@ -49,32 +49,34 @@ impl ActiveEnum {
                 for meta in list.iter() {
                     if let Meta::NameValue(nv) = meta {
                         if let Some(name) = nv.path.get_ident() {
-                            if name == "rs_type" {
-                                if let Lit::Str(litstr) = &nv.lit {
-                                    rs_type = syn::parse_str::<TokenStream>(&litstr.value())
-                                        .map_err(Error::Syn);
-                                }
-                            } else if name == "db_type" {
-                                if let Lit::Str(litstr) = &nv.lit {
-                                    let s = litstr.value();
-                                    match s.as_ref() {
-                                        "Enum" => {
-                                            db_type = Ok(quote! {
-                                                Enum {
-                                                    name: Self::name(),
-                                                    variants: Self::iden_values(),
-                                                }
-                                            })
-                                        }
-                                        _ => {
-                                            db_type = syn::parse_str::<TokenStream>(&s)
-                                                .map_err(Error::Syn);
+                            if let Expr::Lit(exprlit) = &nv.value{
+                                if name == "rs_type" {
+                                    if let Lit::Str(litstr) = &exprlit.lit {
+                                        rs_type = syn::parse_str::<TokenStream>(&litstr.value())
+                                            .map_err(Error::Syn);
+                                    }
+                                } else if name == "db_type" {
+                                    if let Lit::Str(litstr) = &exprlit.lit {
+                                        let s = litstr.value();
+                                        match s.as_ref() {
+                                            "Enum" => {
+                                                db_type = Ok(quote! {
+                                                    Enum {
+                                                        name: Self::name(),
+                                                        variants: Self::iden_values(),
+                                                    }
+                                                })
+                                            }
+                                            _ => {
+                                                db_type = syn::parse_str::<TokenStream>(&s)
+                                                    .map_err(Error::Syn);
+                                            }
```

Comment:
This part doesn't work as expected. Please check how the original PR made the changes https://github.com/SeaQL/sea-orm/pull/1559